pkg/config/config.go

```diff
@@ -152,6 +152,15 @@ func (bp Blueprint) GroupIndex(groupName GroupName) int {
 	return -1
 }
 
+// Group returns the deployment group with a given name
+func (bp Blueprint) Group(groupName GroupName) (DeploymentGroup, error) {
```

Comment:
`s/groupName/n/` it's too verbose and "ungoolangy"

Reply:
Addressed here and in `GroupIndex` since the type now conveys the argument sufficiently.